src/plugins/vis_type_timeseries/common/vis_schema.ts

```diff
@@ -7,7 +7,7 @@
  */
 
 import { schema } from '@kbn/config-schema';
-import { TypeOptions } from '@kbn/config-schema/target/types/types';
+import { TypeOptions } from '@kbn/config-schema/target/types';
```

Comment:
Can you elaborate on why this change is necessary?

Reply:
I've standardised the build and both types and transpiled code are in the same output folder https://github.com/elastic/kibana/pull/96273/files#diff-967ec928a0bb7993de410d4dd6d23c1bac6648fe8acde693f9aa5b9df217216cL6

Reply:
@flash1293 is that okay?